client/src/components/game/phases/Events.vue

```diff
@@ -30,6 +33,24 @@
           </div>
         </div>
       </div>
+      <div class="active-accomplishments col-3" 
+        v-if="currentEvent.clientViewHandler == 'INFLUENCES_SELECT' ||
```

Comment:
Could we push this conditional logic into the `Events` Vue class instead? Like a getter `shouldShowAccomplishments` or something along those lines?